exercises/go-counting/canonical-data.json

```diff
@@ -125,9 +125,9 @@
 		],
       },
       "expected": {
-        "TerritoryBlack": []
-        "TerritoryWhite": []
-	"TerritoryNone": [(0, 0)]
+        "territoryBlack": [],
+        "territoryWhite": [],
+	"territoryNone": [[0, 0]]
```

Comment:
Watch the indents on the `"territoryNone":`'s.  they should be the same as the two above.

Reply:
Is this why it is not valid JSON? Or is there some other problem?